app/views/provider/admin/dashboards/_apiap_navigation.html.slim

```diff
@@ -0,0 +1,27 @@
+nav.DashboardNavigation.DashboardNavigation-tabs
+  ul.DashboardNavigation-list
+    li.DashboardNavigation-list-item
+      = dashboard_apiap_tab_label 'tab-products', 'Product', @services, icon_name: 'puzzle-piece', current_tab: true
+    li.DashboardNavigation-list-item
+      = dashboard_apiap_tab_label 'tab-backends', 'Backend', current_account.backend_apis, icon_name: 'database'
+
+javascript:
```

Comment:
Can we put that in a file instead of embedding the javascript?

Reply:
Would be better as asset or pack?